iwyu_globals.cc

```diff
@@ -90,9 +90,15 @@ static void PrintHelp(const char* extra_msg) {
          "        Note that this only affects comments and alignment thereof,\n"
          "        the maximum line length can still be exceeded with long\n"
```

Comment:
We need to get the command-line arg changes into the manpage as well (https://github.com/include-what-you-use/include-what-you-use/blob/master/include-what-you-use.1). Let me know if you don't have a good way to test that (or the patience to do it :)), and I can try and push a patch when we've settled on the command-line design.